src/containers/Console.js

```diff
@@ -23,6 +27,11 @@ function mapStateToProps(state) {
 
 function mapDispatchToProps(dispatch) {
   return {
+    onClearConsoleEntries(e) {
+      e.stopPropagation();
```

Comment:
So `stopPropagation()` is the right place to do, but this isn’t an ideal place to do it—containers shouldn’t know anything about the UI that’s made by the presentational components they wrap. Here we are leaking the knowledge of what actual UI would trigger the `clearConsoleEntries()` callback.

So, we should do the `stopPropagation` inside the presentational component. A couple of options:

1. Use an arrow function for the `onClick`, which stops propagation and calls back to `onClearConsoleEntries`. You’d also have to remove the `jsx-no-bind` eslint rule, which I think is basically fine, since it’s based on a misunderstanding of performance characteristics anyway.
2. Change `<Console>` into a class component and define a `_handleClickClearButton` method which is then given as the `onClick`. This has the advantage of maintaining our purity with respect to `jsx-no-bind`, but little else.

Reply:
Will go with option 1 :)